README.md

```diff
@@ -102,26 +102,34 @@ To set up **MacOS** for native development, complete the following steps:
 ### Docker
 
 For production hosting and deployment with **Docker**, use the `Dockerfile` and
-`docker-compose.yml` in the project root directory. To create a local
-development environment with **Docker**, complete the following steps:
+`docker-compose.yml` in the project root directory.
 
-- Install Docker Desktop
-- Run `docker compose -f .devcontainer/docker-compose.yml up -d`
-- Run `docker compose -f .devcontainer/docker-compose.yml exec app bin/setup`
-- Finally, run `docker compose -f .devcontainer/docker-compose.yml exec app bin/dev`
+For local development, install and launch [Docker Desktop], and run:
```

Comment:
Docker Desktop might not be needed here, this works fine with only Docker Engine on linux, or any alternative (like orbstack)